editor/gui/editor_quick_open_dialog.cpp

```diff
@@ -932,43 +933,42 @@ static Vector2i _get_name_interval(const Vector2i &p_interval, int p_dir_index)
 
 QuickOpenResultListItem::QuickOpenResultListItem() {
 	set_h_size_flags(Control::SIZE_EXPAND_FILL);
-	add_theme_constant_override("separation", 4 * EDSCALE);
+	begin_bulk_theme_override();
+	add_theme_constant_override("margin_left", 6 * EDSCALE);
+	add_theme_constant_override("margin_right", 6 * EDSCALE);
+	end_bulk_theme_override();
```

Comment:
If I remember correctly, using bulk functions is necessary only for runtime changed inside tree. So there is no need to use it in constructor. Same for `QuickOpenResultGridItem` constructor below.